e2e_playwright/st_dataframe_interactions.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
I'm slightly worried about the positions that are hardcoded here especially if dataframe changes occur or just flakiness. I can't think of a better way to resolve this so I guess I'm ok with this for now but we should definitely try to monitor these...

Reply:
> I'm slightly worried about the positions

Yep indeed :( But I wasn't able to figure out any other way since there aren't any elements to pin-point on a Canvas. I'm not sure if there is any other solution for testing interactivity with canvas.